### PR TITLE
Feat hobbygroups

### PR DESCRIPTION
**Pull request by martinlunde**
## What kind of a pull request is this?
Feature request. Added Hobbygroups to ow4, and solved issue #1788?
- [x] QA / Code Review

## Code Checklist

- [x] The code follows dotkom code style 
- [x] The code passes the defined tests
- [ ] I have added tests for the code I added
- [ ] I have provided documentation for the code I added
- [x] The code is ready to be merged

## (Possible) Breaking Changes

- [ ] The changes are backwards compatible

- [ ] I have updated the build configuration
- [ ] These changes requires changes to configuration in production 
    - [ ] I have applied the required changes in production
    - [ ] I cannot apply the required changes in production before this is deployed.

## Screenshot(s) if appropriate
Before:
skjermbilde 2017-04-19 kl 23 30 02

After:
skjermbilde 2017-04-20 kl 13 07 52

### CODE REVIEW

**Comment by OptimusCrime** [comment]
Does it make more sense to have between Ressurser and Wiki? It kind of "belongs" more to that part  (going from most essential to most external from left to right). My 5 cents.

File: assets/resourcecenter/less/resourcecenter.less [hunk 1]
```diff
@@ -2,7 +2,7 @@
 /* ResourceCenter */
 
 #resourcecenter {
-    .resourcedescription {
+    .hobbydescription {
```
Comment (sklirg) [inline assets/resourcecenter/less/resourcecenter.less]: Any reason for this name change? 
* If they are going to have different styles, use different names. 
* If it's the same style across both, create a common name.

File: templates/base.html [hunk 2]
```diff
@@ -51,6 +51,7 @@
                 <li><a href="{% url 'resourcecenter_index' %}">Ressurser</a></li>
                 <li><a href="/wiki/">Wiki</a></li>
                 <li><a href="/webshop/">Webshop</a></li>
+                <li><a href="/hobbygroups/" class="hidden-sm">Interessegrupper</a></li>
```
Comment (sklirg) [inline templates/base.html]: Use the `{% url %}` syntax from Django to generate the URL to the [index page](https://github.com/dotkom/onlineweb4/pull/1847/files#diff-a92964c7df7035eb70f7fffb04eb5615) of the app. (Tip: Look at how it's done for resourcecenter 3 lines above 😉 )

File: templates/hobbygroups/index.html [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+			</div>
+		</div>
+
+        <div class="row">
+			<div class="col-xs-12 col-sm-6 col-md-6">
+				<div class="col-md-5">
+					<img src="{{ STATIC_URL }}img/hobbygroups/strikk.png" alt="Strikk og Drikk" />
+				</div>
+				<div class="col-md-7 hobbydescription">
+					<h3>Strikk & drikk</h3>
```
Comment (sklirg) [inline templates/hobbygroups/index.html]: Use `&amp;` instead, because some browsers might try to parse `&` and unexpected results might occur.

**Comment by sklirg** [comment]
Also: By having an HTML file containing all the information about all the hobby groups, dotkom would have to be contacted every time something changes or a group is added. This is tiresome. Maybe look into creating some models for the hobby groups? You might even consider looking into #1209 at the same time.

**Comment by OptimusCrime** [comment]
Btw, before and after images are mixed, unless you removed the menu item in this PR.

File: assets/resourcecenter/less/resourcecenter.less (hunk 1 above)
Comment (martinlunde) [inline assets/resourcecenter/less/resourcecenter.less]: fixed, refactor chose to rename a bit more than it was supposed to

File: templates/base.html (hunk 2 above)
Comment (martinlunde) [inline templates/base.html]: fixed

File: templates/hobbygroups/index.html (hunk 3 above)
Comment (martinlunde) [inline templates/hobbygroups/index.html]: fixed when moving to models

File: apps/hobbygroups/models.py [hunk 4]
```diff
@@ -0,0 +1,12 @@
+from django.db import models
+from apps.hobbygroups import settings
+
+
+class Hobby(models.Model):
+    title = models.CharField(max_length=25)
+    description = models.CharField(max_length=250)
+    image = models.ImageField(upload_to=settings.IMAGES_PATH)
+    read_more_link = models.CharField(max_length=100, blank=True)
```
Comment (sklirg) [inline apps/hobbygroups/models.py]: This could probably be set as an URL field to gain the benefits of URL validation :-)

**Pull request merged**